requirements/requirements.txt

```diff
@@ -327,7 +327,7 @@ requests-oauthlib==1.3.0
     #   kubernetes
     #   msrest
     #   social-auth-core
-rsa==4.0
+rsa==4.7
```

Comment:
There is a 4.7.2 released (even a 4.8 release). This is quite a few minor version jumps. I guess `google-auth` requires this newer version ?

Reply:
That's weird... I just ran updater.sh and it didn't change this line, but when I change it manually and re-run `make requirements`, it works. I will commit and push that.